File: c/fizzbuzz.c
#include <stdio.h>

void IO_fizzBuzz();

void main_nu() {
	IO_fizzBuzz();
}

void IO_fizzBuzz() {

	for (int x = 1; x <= 100; x++) {


		if (0 == x % 3 && 0 == x % 5)
			printf("FizzBuzz");
		else if (0 == x % 3)
			printf("Fizz");
		else if (0 == x % 5)
			printf("Buzz");
		else printf("%d", x);

		printf("\n");

	}
}

File: julia/fizzbuzz.jl
FizzBuzz(minim,maxim) =
    for number in minim:maxim
        if number % 3 == 0
            print("Fizz")
        end
        if number % 5 == 0
            print("Buzz")
        end
        if (number % 5 != 0) & (number % 3 !=0)
            print(number)
        end
        print("\n")
    end

FizzBuzz(1,100)
